pkg/printers/internalversion/printers_test.go

```diff
@@ -6206,6 +6208,322 @@ func TestPrintFlowSchema(t *testing.T) {
 	}
 }
 
+func TestPrintDeviceClass(t *testing.T) {
+
+	tests := []struct {
+		deviceClass resourceapis.DeviceClass
+		expected    []metav1.TableRow
+	}{
+		{
+			deviceClass: resourceapis.DeviceClass{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "deviceclass",
```

Comment:
Could you please set here something other than `deviceclass` to not confuse with the resource name?

Reply:
I have added the "test-" prefix to all the cases.